GUIDE.md

```diff
@@ -0,0 +1,165 @@
+# Assisted Installer Unofficial Install Guide
+
+## Prerequisites
+
+1. Install CentOS8/RHEL8 on the assisted installer host<sup id="a1">[1](#f1)</sup>
+   - This host will run minikube and the UI for deploying OpenShift on Bare Metal
+1. Setup DHCP/DNS records for the following OpenShift nodes and VIPs. List includes
+   - Master nodes
+   - Worker nodes
+   - API VIP
+   - Ingress VIP
+     > NOTE: dnsmasq can be used to setup [DHCP](https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#creating-dhcp-reservations-using-dnsmasq-option2_ipi-install-prerequisites) and [DNS](https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#creating-dns-records-using-dnsmasq-option2_ipi-install-prerequisites). Out of scope for this document but I've included links on how to do it.
+1. Ensure these values are properly set via nslookup and dig commands from your assisted installer host
+1. Download your Pull Secret from https://cloud.redhat.com/openshift/install/metal/user-provisioned
+
+## Deployment
+
+Procedure
+
+As `$USER` user with sudo privileges ,
+
+1.  Generate an SSH key
+
+        [$USER@assisted_installer ~]# ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ''
```

Comment:
in test-infra u have have folder with ssh keys. so u can just use them
ssh_key/key

Reply:
@tsorya @iranzo However,  if you have a folder with ssh keys, there is an assumption that ssh keys exist. Likely a better approach here would be to say generate an ssh key if not available.